Tasks/DotNetCoreCLIV2/Tests/DotnetMockHelper.ts

```diff
@@ -66,7 +65,7 @@ export class DotnetMockHelper {
     }
 
     public registerNugetUtilityMock(projectFile: string[]) {
-        this.tmr.registerMock('nuget-task-common/Utility', {
+        this.tmr.registerMock('packaging-common/nuget/Utility', {
             getPatternsArrayFromInput: function(input) {
                 return [`fromMockedUtility-${input}`];
             },
```

Comment:
getNuGetFeedRegistryUrl below doesn't match current parameter order.

Reply:
I plan to move these to a common location too, as soon as I get a chance :)